### PR TITLE
Add inclusive language guidelines

### PR DESCRIPTION
**Pull request by joseluisbolos**
This PR adds new language guidelines for inclusive, non-discriminatory language.

I would really appreciate if @hollsk and @typothecary could have a look at it. ;)

### CODE REVIEW

File: practices/language.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Stereotyping means presuming a range of things about people based on one or two of their personal characteristics such as their appearance, apparent intelligence, personality or character, or their gender, sexual orientation, race, ethnicity, age, location, socioeconomic status or disability. We don't use language that reinforces stereotypes.
+
+
+## Examples
+
+
+### Genderless language
+
+* When writing documentation or examples that don't refer to specific people, we prefer the gender neutral honorific [`Mx`](http://www.oxforddictionaries.com/definition/english/mx#nav2).
```
Comment (hollsk) [inline practices/language.md]: R.e. Mx: This is wonderful. I don't think it should be in our documentation, though :D 

- It's a self-selected designation, and definitely not something that should be assigned to an individual without their permission (no different to referring to a woman as "Mr" or to a man as "Miss")
- It's a title, and honestly there isn't much of a use-case for titles in technical documentation unless it's documenting how to directly address users

File: practices/language.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* We avoid examples that use _bro_ language, even when using verbal communication.
+
+Good:
+> Server: Hello.
+> Client: Hey server, I have a message for you.
+
+Bad:
+> Server: Hello.
+> Client: Hey _bro_, I have a message for you.
+
```
Comment (hollsk) [inline practices/language.md]: NoBros: Wonderful. Might also want to add a "guys" example?

**Comment by hollsk** [comment]
I only have a couple of comments and aside from those two comments I would like to give you a GIGANTIC THUMBS-UP, even bigger than the measly little one that Github provides, so please increase your screen magnification by 10: 👍

**Comment by dotcode** [comment]
Addresses #32, to a large degree.

**Pull request merged**